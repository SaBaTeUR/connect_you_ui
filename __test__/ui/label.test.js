import React from 'react'
import Label from '../../src/ui/label/label.jsx'
import renderer from 'react-test-renderer'

describe('UI testing...', () => {
  it('First Test', () => {
    const component = renderer.create(<Label></Label>)
    expect(component.toJSON()).toMatchSnapshot()
  })
});