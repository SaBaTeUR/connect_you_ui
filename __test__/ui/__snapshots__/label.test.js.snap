// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI testing... First Test 1`] = `
<div
  className="label"
>
  <div
    className="label__header"
  />
  <div
    className="label__text"
  />
</div>
`;
